pymc3/tests/test_sampling.py

```diff
@@ -203,13 +221,24 @@ def test_return_inferencedata(self):
             assert result.posterior.sizes["chain"] == 2
             assert len(result._groups_warmup) > 0
 
-            # inferencedata without tuning
-            result = pm.sample(**kwargs, return_inferencedata=True, discard_tuned_samples=True)
+            # inferencedata without tuning, with prior
+            prior = pm.sample_prior_predictive()
+            result = pm.sample(
+                **kwargs,
+                return_inferencedata=True,
+                discard_tuned_samples=True,
+                idata_kwargs={"prior": prior}
+            )
             assert isinstance(result, az.InferenceData)
             assert result.posterior.sizes["draw"] == 100
             assert result.posterior.sizes["chain"] == 2
             assert len(result._groups_warmup) == 0
-        pass
```

Comment:
unnecessary `pass` statement

Reply:
I like to put them there to mark the end of a test/function (just like an unnecessary `return`).

In my opinion they have a few advantages:
+ trailing comments are collapsed together with the code
+ because the `pass` in a test, or `return` in a function was put there on purpose, you don't wonder if somebody just stopped writing, or accidentally deleted some code

Not saying you must put it back in - just showing a maybe new perspective :)

Reply:
To me that looks weird and confusing.